es-vue-base/src/lib-components/EsFileInput.vue

```diff
@@ -0,0 +1,267 @@
+<template>
+    <div
+        role="button"
+        tabindex="0"
+        class="es-file-upload align-items-center rounded d-flex flex-column justify-content-center p-2"
+        :class="{ 'active': active }"
+        @dragenter.stop.prevent="active = true"
+        @dragover.stop.prevent="active = true"
+        @dragleave.stop.prevent="active = false"
+        @dragend.stop.prevent="onDrop"
+        @drop.stop.prevent="onDrop"
+        @click.self="openFilePicker"
+        @keypress.enter="openFilePicker">
```

Comment:
Nit: Usually `space` is considered a click trigger on a focused element as well as enter.